tests/mgr/test_cephmgr_throttling.py

```diff
@@ -0,0 +1,40 @@
+from cli.cephadm.cephadm import CephAdm
+
+
+class CephmgrThrottlingError(Exception):
+    pass
+
+
+def run(ceph_cluster, **kw):
+    """Verify cephmgr throttling
+    Args:
+        **kw: Key/value pairs of configuration information to be used in the test.
+        e.g
+        test:
+            name: Verify cephmgr throttling
+            desc: Verify cephmgr throttling
+            polarion-id: CEPH-83573248
+            module: test_cephmgr_throttling
+            config:
+                operation: enable
+                module: balancer
+                key: target_max_misplaced_ratio
+                value: .07
+    """
+    node = ceph_cluster.get_nodes(role="mon")[0]
+    action = kw.get("config").get("operation")
```

Comment:
Add validations for these config parameters

Reply:
Thanks @vamahaja 
Updated